File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/TEstado.vb
Imports System.Collections.Generic

Public Class TEstado
    Public Shared Function Add(value As Models.TEstado) As Integer
        Return Data.TEstado.Add(value)
    End Function

    Public Shared Function Edit(value As Models.TEstado) As Integer
        Return Data.TEstado.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.TEstado.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.TEstado
        Return Data.TEstado.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.TEstado
        Return Data.TEstado.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.TEstado)
        Return Data.TEstado.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171704/Business/TEstado.cs
using System.Collections.Generic;

namespace Business
{
    public class TEstado
    {
	    private static Data.TEstado obj = new Data.TEstado();

        public static int Add(Models.TEstado value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.TEstado value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.TEstado GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.TEstado GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.TEstado> GetAll()
        {
            return obj.GetAll();
        }
    }
}